assessments/projects/jaeger/self-assessment.md

```diff
@@ -0,0 +1,317 @@
+
+* Jaeger is capable of encrypting data using Transport Layer Security (TLS) in conjunction with mutual TLS (mTLS). Using mTLS offers a better level of security because it necessitates the validity of certificates on both the client and the server (and thus mitigates man-in-the middle attacks). 
+* The OpenTelemetry SDK can communicate to the jaeger collector by gRPC or HTTP with the option to enable TLS with mTLS.
+* The Jaeger Collector, Ingester and Query can communicate to storage via third-party software like Cassandra, ElasticSearch and Kafka all with TLS with mTLS support. 
+* Elasticsearch supports bearer token propagation and Kafka also supports Kerberos and plaintext authentication.
+
+
+### Security Relevant.  
+
+#### Authentication and Authorization
+* Bearer tokens are an option offered by Jaeger for these purposes.
+* Users and apps are granted restricted access to Jaeger's capabilities through the use of OAuth2 tokens. Jaeger also supports plaintext and Kerberos authentication.
+
+#### Access Control: 
+* Role-based access control (RBAC) is a feature that Jaeger provides for both users and applications. 
+* This enables system administrators to specify the roles and permissions of various users. This guarantees that sensitive information and functions can only be accessed by authorized users.
+
+#### Security Auditing:
+* Jaeger provides tools for monitoring and auditing. It has the ability to monitor user actions such logins, searches, and sensitive data access. Administrators can utilize this data to quickly identify and fix possible security issues.
+
+## Project compliance
+
+* Compliance.  List any security standards or sub-sections the project is
+  already documented as meeting (PCI-DSS, COBIT, ISO, GDPR, etc.).
+
+Jaeger does not currently document meeting particular compliance standards.
+
+## Secure development practices
+
+### Development Pipeline.  
+
+* To contribute change, an issue must be opened first. The issue should describe:
+  * Requirement: what use case is being solved
+  * Problem: what is missing from Jaeger to solve the requirement
+  * Proposal: what changes are going to be made to solve the problem 
+* All packages must have a complementary test to go along with it.
+* After the approach is agreed upon, code changes can be made and a pull request can be opened
+* If one wants to work on an existing issue, they can leave a comment under the issue and submit a pull request
+* Create a new local branch and sign all your commits
+* All dependencies are automatically checked and updated using dependabot
+* Push your changes and look for an output containing an URL to create a pull request
+* Each PR should have a contain:
+  * <50 characters
+  * Capitalized title
+  * Title not end with a period
+  * A description of the problem it’s solving or a reference to the corresponding issue
+  * Summary of what changes were made
+* The pull request will then be reviewed and merged by the maintainer
```

Comment:
Please elaborate on the process that happens henceforth. 
- Is review and subsequent approval mandated for every PR?
- What does the CI pipeline look like? what are the security checks performed in this pipeline (SAST, SCA, secret scan, container image scan etc.)